APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,212 @@
+"Astropy Project".  That umbrella term applies to both coordinated and
+affiliated packages, as well as the overall community.
+
+
+A criteria-based package review process
+=======================================
+
+This APE also proposes a second change to the review process, motivated by the
+concerns in the "Motivation" section above about the committee's review
+process. To counter these concerns, prior to this APE the coordination committee
+has informally adopted a checklist-based approach, checking if certain "bullet
+points" are met.  While the formal criteria are listed in relatively vague terms
+on the affiliated package web site, the details of how those criteria are met
+have been fairly consistent using an unwritten but fairly standard list. In
+practice, this has meant that affiliated packages are given "provisional" status
+if some of these items are checked off are met, or "full" if a larger set of
+them are.
+
+This de facto checklist implies that a more transparent and understandable
+policy would be to make this checklist formal and completely public, and use it
+directly as the formal review criteria - i.e., a rubric.  These criteria will be
+laid out explicitly in a document, with various levels defined for each criteria
+(generally corresponding to "good", "acceptable", and "unacceptable"). This
+will make the review process much faster and (eventually) open the review
+process beyond the coordination committee, thereby address the concerns from the
+"Motivation" section.
+
+
+A community review process for affiliated packages
+==================================================
+
+On a longer term horizon, a criteria-based review checklist will allow the
+review process to change to a more "peer-review" model.  That is, instead of
+every package being vetted by the coordination committee, the committee can act
+more as "editors", sometimes offering the review opportunity to other members of
+the Astropy community.  While the details of this process will not be outlined
+here, as they by necessity should be flexible enough to accomodate the
+experiences of the first few reviewers, this APE sets this as the eventual
+*goal* for the affiliated package review process.
+
+
+Branches and pull requests
+--------------------------
+
+N/A
+
+
+Implementation
+--------------
+
+The changes outlined in this APE will be implemented as three distinct efforts:
+
+* The new package review criteria have already been implemented by the
+  coordination committee and reviewed by the community (Available in Google doc
+  form `here <https://docs.google.com/document/d/15PJf2PROXMa7xwTDvWnjXB_9KNuO2Ia4_kkxJ7MPazE/edit?usp=sharing>`_.
+  These criteria will need to be translated to HTML and put on the affiliated
+  package instructions section of the Astropy website.  The affiliated package
+  listing page will also need to be updated for these new criteria - a concept
+  for this new site is `viewable here <example_affilpkg_page.png>`_.
```

Comment:
Could you remind us where the data are that was used to construct example_affilpkg_page.png?  Is that example meant to be representative of actual package status, or was it just created using random data?  For example, I'm pretty sure that PyDL is Python 3 compatible.

Reply:
I think the data for that example might have been *entirely* made-up.  That is, it's a mock up of what it *might* look like. The authoritative source for this is http://www.astropy.org/affiliated/registry.json , and you can see there that `PyDL` is "good" for Python3, so nothing to worry about there!

Reply:
Good, thank you for checking.  Unless there are some privacy concerns that I am missing, maybe the best thing is actually to use the real data.  I think the graphics look good overall.  I'm guessing the title of the graph was also made up.

Reply:
The main problem is that this was a "mock up" and I believe it would take quite a bit of work to do a "real" version (and it wouldn't exercise all the features).  But maybe @granttremblay can confirm on this? (he mad the mockup a while back)

Reply:
Can we add a banner of text in the screenshot that says 'not actual data' or something like this?

Reply:
Done re: banner - hopefully that will also make it a bit clearer if anyone else looks and sees things like @weaverba137 did!